### PR TITLE
Add not-null constraints to a couple of places

### PR DESCRIPTION
**Pull request by gzsombor**
Add not null constraints, where it is deserved

### CODE REVIEW

**Comment by jdubois** [comment]
I'm currently fighting with Liquibase to have the "diff" working correctly, and I'm heavily changing those files -> I will have a look later, but I'm not merging it as I already know it will not work!

**Comment by gzsombor** [comment]
I don't understand, what wont work ? I think, both the 'login' in the user table, and the user_id in the token table shouldn't be null. Yes, if you try to re-generate your system with the new code, this will cause some problems, but it wont be the first case, as compatibility is not a priority as I saw.

**Comment by jdubois** [comment]
OK, I will merge this first, and then I'll have a try when with my code

**Comment by gzsombor** [comment]
Thanks !

**Comment by jdubois** [comment]
I'm currently merging this with my code (I can't push it now, I depend on snapshot versions of Liquibase)

File: app/templates/src/main/resources/config/liquibase/changelog/_initial_schema.xml [hunk 1]
```diff
@@ -77,7 +77,9 @@
             <column name="series" type="varchar(255)">
                 <constraints primaryKey="true" nullable="false"/>
             </column>
-            <column name="user_id" type="bigint"/>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false" />
```
Comment (jdubois) [inline app/templates/src/main/resources/config/liquibase/changelog/_initial_schema.xml]: I am at least dropping this, as I can't merge it correctly at the moment

**Pull request merged**